<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: white;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgroundHeader">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>831</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: #A3A3A3; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 16px;
font: bold;
font-family: Comic Neue;


}</string>
    </property>
    <property name="text">
     <string>Adivinhe o Carro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imagemLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/images/cuatroDesarrolladores.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tentativas">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 16px;
font: bold;
font-family: Comic Neue;

}</string>
    </property>
    <property name="text">
     <string>Tentativas: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputCarro">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius: 5px;
border: 1px solid black;
color: black;
placeholderText-color: black;

}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Escreva aqui </string>
    </property>
   </widget>
   <widget class="QLabel" name="imagemCarro">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/images/Audi RS7.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="desistir">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 5px;
color: white;
background-color: #E53B3B;
font: bold;
font-size: 12px;

}</string>
    </property>
    <property name="text">
     <string>Desistir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 5px;
color: white;
background-color: #90E53B;
font: bold;
font-size: 12px;

}</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QLabel" name="tabelaDicas">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>221</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid black;
border-radius: 5px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imagemSetinha">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/images/voltar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="voltar">
    <property name="geometry">
     <rect>
      <x>734</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font-size: 14px;
font: bold;
border: none;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imagemDica">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../public/images/dica.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDica">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 16px;

}</string>
    </property>
    <property name="text">
     <string>Dicas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>181</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="anoLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 14px;

}</string>
       </property>
       <property name="text">
        <string>Ano: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="motorLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 14px;

}</string>
       </property>
       <property name="text">
        <string>Motor: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="potenciaLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 14px;

}</string>
       </property>
       <property name="text">
        <string>Potência: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tracaoLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 14px;

}</string>
       </property>
       <property name="text">
        <string>Tração: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushDica">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: none;
color: none;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
